composition_example/static/basic.html

```diff
@@ -1,12 +1,13 @@
 <html>
   <head>
-    <link rel="stylesheet" href="/static/basic.css">
+    <link rel="stylesheet" href="/static/head/basic.css">
     <link uic-remove rel="stylesheet" href="basic.css">
   </head>
   <body>
     <h1 uic-remove>Basic Web Elements</h1>
     <h3 uic-remove>The Header</h3>
     <uic-fragment name="header">
+      <link rel="stylesheet" href="/static/body/basic.css#header">
```

Comment:
would be good to also have on testcase with two or more stylesheets in a fragment.